stats/src/main/java/com/facebook/stats/mx/StatsReader.java

```diff
@@ -36,6 +36,8 @@
   public MultiWindowDistribution getDistribution(String key);
   public String getAttribute(StatType key);
   public String getAttribute(String key);
+  public Callable<Long> getDynamicCounter(StatType key);
+  public Callable<Long> getDynamicCounter(String key);
```

Comment:
and i don't think the stats object should be used as some sort of global map to share callables across